docker/cpu.Dockerfile

```diff
@@ -29,6 +29,6 @@ ADD ./setup.py ./setup.py
 ADD ./README.md ./README.md
 ADD ./MANIFEST.in ./MANIFEST.in
 ADD ./kaggle_environments ./kaggle_environments
-RUN pip install Flask bitsandbytes accelerate vec-noise chess && pip install . && pytest
```

Comment:
Intentional? Surprising since this if for the Chess env

Reply:
yes this should be installed with kaggle_environments dependencies here: https://github.com/Kaggle/kaggle-environments/blob/master/setup.py#L49